tasks/release.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "Tito release"
+  command: "tito release {{ releasers | join(' ') }}"
+  args:
+    chdir: "packages/{{ inventory_hostname }}"
+
+- name: "Wait for tasks"
```

Comment:
Couldn't this have been factored out to a separate file and included here?

Reply:
Good Eye!  It definitely could, and probably should.  With the direction shift to utilizing obal, these ansible playbooks should be deprecated shortly though.  This PR is to just get what I've been utilizing locally as a stop gap into this repo before the shift happened.